js/models/vehicle.ts

```diff
@@ -50,3 +50,10 @@ export const vehicleFromVehicleData = (vehicleData: VehicleData): Vehicle => {
     },
   };
 };
+
+export const estimatedArrivalOfVehicle = (vehicle: Vehicle) => {
```

Comment:
observation: ha, I did this too. https://github.com/mbta/orbit/pull/378/files#diff-38050556722f95612a65c8d9f25ffda52df105a5c1e14a989b83724d8559acb0R67

Not sure how much the array bounds check matters?

Reply:
The main difference is that yours returns explicit `null`, whereas mine is less specific and can return `undefined`. So I think I prefer yours. Since that branch is about to go in, I'll wait and rebase.